test/record/record-requestSpec.js

```diff
@@ -75,11 +75,12 @@ describe( 'records are requested from cache and storage sequentually', function(
 		expect( options.storage.lastRequestedKey ).toBe( 'doesNotExist' );
 	});
 
-	it( 'fails gracefully if an error occured out of order', function( done ){
+	//TODO: yasserf: I don't get this test
```

Comment:
I meant more that done is called in the record request and and after 20 milliseconds, which means it can exit successfully without any expectations run